src/apicast-cli/cmd/rockspec/generator.lua

```diff
@@ -0,0 +1,80 @@
+local persist = require "luarocks.persist"
+local type_check = require "luarocks.type_check"
+
+local _M = {}
+
+
+local function configure_lua_version(rockspec, luaver)
```

Comment:
We don't really care about this. Actually this should be nil as we don't run on lua, but openresty.